src/components/designer/lightning/connect/LndConnect.tsx

```diff
@@ -21,16 +21,28 @@ const LndConnect: React.FC<Props> = ({ node }) => {
     const { tlsCert, adminMacaroon, invoiceMacaroon, readonlyMacaroon } = node.paths;
     try {
       const host = `127.0.0.1:${node.ports.grpc}`;
+      const resthost = `127.0.0.1:${node.ports.rest}`;
+
       const cert = await read(tlsCert);
       const adminMac = await read(adminMacaroon, 'hex');
       const invoiceMac = await read(invoiceMacaroon, 'hex');
       const readonlyMac = await read(readonlyMacaroon, 'hex');
 
       const values: Record<string, string> = {};
-      values[l('adminUrl')] = encode({ host, cert, macaroon: adminMac });
-      values[l('invoiceUrl')] = encode({ host, cert, macaroon: invoiceMac });
-      values[l('readOnlyUrl')] = encode({ host, cert, macaroon: readonlyMac });
-
+      values[l('GRPCadminUrl')] = encode({ host, cert, macaroon: adminMac });
```

Comment:
Just a small nitpick, can you change these keys to camel case. ex: `grpcAdminUrl`, `restInvoiceUrl`, etc.

Reply:
ok I will do that